tests/index.scss

```diff
@@ -0,0 +1,13 @@
+$use-embedded-font: true;
```

Comment:
This was just for convenience as I had to debug stuff directly in Karma Debugger, once you do that some CSS helps here.